tfhe/src/integer/server_key/comparator.rs

```diff
@@ -251,12 +232,61 @@ impl<'a> Comparator<'a> {
     /// Reduces a vec containing shortint blocks that encrypts a sign
     /// (inferior, equal, superior) to one single shortint block containing the
     /// final sign
-    fn reduce_signs_parallelized(
+    fn reduce_signs<F>(
         &self,
         mut sign_blocks: Vec<crate::shortint::Ciphertext>,
-    ) -> crate::shortint::Ciphertext {
+        sign_result_handler_fn: F,
+    ) -> crate::shortint::Ciphertext
+    where
+        F: Fn(u64) -> u64,
+    {
+        while sign_blocks.len() != 2 {
```

Comment:
should this be > 2 as in the function below ?

Reply:
hum yes